packages/dds/map/src/test/mocha/directoryEquivalenceUtils.ts

```diff
@@ -28,14 +35,40 @@ function assertEventualConsistencyCore(
 
 	// Check key/value pairs in both directories.
 	for (const key of first.keys()) {
-		assert.strictEqual(
-			first.get(key),
-			second.get(key),
-			`Key not found or value not matching ` +
-				`key: ${key}, value in dir first at path ${first.absolutePath}: ${first.get(
-					key,
-				)} and in second at path ${second.absolutePath}: ${second.get(key)}`,
-		);
+		const firstVal: unknown = first.get(key);
+		const secondVal: unknown = second.get(key);
+		const firstObj: FluidObject<IFluidHandle> = firstVal as FluidObject<IFluidHandle>;
```

Comment:
this should be very similar to the map function. the `as FluidObject<IFluidHandle>` should be unnecessary if you check if they are objects first

Reply:
I still got lint errors without the cast after checking that they are objects, so all of the values are still casted